### PR TITLE
#270 Selectize Modifications For Pasting Delimited Values

### PR DESCRIPTION
**Pull request by jhash**
Simple fix for adding paste support where values are delimited by whatever regex or characters desired - does not interfere with existing delimiter functionality at all.

### CODE REVIEW

**Comment by brianreavis** [comment]
@jhash Sorry for the delays in getting to this. I'm going to make a few minor changes, but all in all it looks really great. Thanks!

**Comment by brianreavis** [comment]
This is now a part of [0.12.0](https://github.com/brianreavis/selectize.js/releases/tag/v0.12.0) (just released)

**Comment by island205** [comment]
love this feature

File: src/defaults.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ Selectize.count = 0;
 Selectize.defaults = {
 	plugins: [],
 	delimiter: ',',
+	splitOn: null, // Regex or string for splitting up values from a paste command
```
Comment (zinkkrysty) [inline src/defaults.js:5]: Is this option missing from documentation? I couldn't find it

**Pull request merged**